### PR TITLE
Added space between two resource strings

### PR DESCRIPTION
**Pull request by NimrodRak**
Fixed issue where there was a space missing between two of the strings in an offline reading message.
Bug: T285065

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt [hunk 1]
```diff
@@ -29,6 +29,7 @@ class LinkPreviewErrorView : LinearLayout {
 
         if (errorType === LinkPreviewErrorType.OFFLINE) {
             val message = (resources.getString(R.string.page_offline_notice_cannot_load_while_offline) +
+                    " " +
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt:32]: Instead of a space, let's make this a newline character (`\n`)

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt (hunk 1 above)
Comment (NimrodRak) [reply app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt:32]: Surely! Will change it by the end of the day.

Comment (dbrant) [reply app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt:32]: No, not really, unless we want to turn it into two TextViews that appear next to each other, but that's not necessary for now.

Comment (NimrodRak) [reply app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt]: Done.

**Pull request merged**